src/gui/redux/api/fetch.js

```diff
@@ -0,0 +1,26 @@
+import { readme, html, docgen } from '../store'
+
+const ownFetch = (reducer, type) => (...fetchArgs) => async (dispatch) => {
+  dispatch(reducer.reset())
+  try {
+    const response = await fetch(...fetchArgs)
+    const text = await response[type]()
+    dispatch(reducer.set(text))
+  } catch (error) {
+    dispatch(reducer.set(''))
```

Comment:
we should manage errors in an other way.
we should add an issue to not forget it or treat it in #108 issue